tawazi/node/node.py

```diff
@@ -77,71 +78,89 @@ class ExecNode:
     When the ExecNode is executed in the DAG, the resulting value will be stored in the `result` instance attribute.
     Note: This class is not meant to be instantiated directly.
         Please use `@xn` decorator.
-    """
 
-    def __init__(
-        self,
-        id_: Identifier,
-        exec_function: Callable[..., Any] = lambda *args, **kwargs: None,
-        args: Optional[List["UsageExecNode"]] = None,
-        kwargs: Optional[Dict[str, "UsageExecNode"]] = None,
-        priority: int = 0,
-        is_sequential: bool = cfg.TAWAZI_IS_SEQUENTIAL,
-        debug: bool = False,
-        tag: Optional[TagOrTags] = None,
-        setup: bool = False,
-        unpack_to: Optional[int] = None,
-        resource: Resource = cfg.TAWAZI_DEFAULT_RESOURCE,
-    ):
-        """Constructor of ExecNode.
 
-        Args:
-            id_ (Identifier): identifier of ExecNode.
-            exec_function (Callable): a callable will be executed in the graph.
-                This is useful to make Joining ExecNodes (Nodes that enforce dependencies on the graph)
-            args (Optional[List[ExecNode]], optional): *args to pass to exec_function.
-            kwargs (Optional[Dict[str, ExecNode]], optional): **kwargs to pass to exec_function.
-            priority (int): priority compared to other ExecNodes; the higher the number the higher the priority.
-            is_sequential (bool): whether to execute this ExecNode in sequential order with respect to others.
-                When this ExecNode must be executed, all other nodes are waited to finish before starting execution.
-                Defaults to False.
-            debug (bool): Make this ExecNode a debug Node. Defaults to False.
-            tag (TagOrTags): Attach a Tag or Tags to this ExecNode. Defaults to None.
-            setup (bool): Make this ExecNode a setup Node. Defaults to False.
-            unpack_to (Optional[int]): if not None, this ExecNode's execution must return unpacked results corresponding
+    Args:
+        id_ (Identifier): identifier of ExecNode.
+        exec_function (Callable): a callable will be executed in the graph.
+        args (Optional[List[ExecNode]], optional): *args to pass to exec_function.
+        kwargs (Optional[Dict[str, ExecNode]], optional): **kwargs to pass to exec_function.
+        priority (int): priority compared to other ExecNodes; the higher the number the higher the priority.
+        is_sequential (bool): whether to execute this ExecNode in sequential order with respect to others.
+            When this ExecNode must be executed, all other nodes are waited to finish before starting execution.
+            Defaults to False.
```

Comment:
I'd say we can remove this line

Reply:
The others have it. Shall we remove them too ?